src/app/components/Timestamp/index.jsx

```diff
@@ -2,15 +2,17 @@ import React from 'react';
 import styled from 'styled-components';
 import { node, string } from 'prop-types';
 import { GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
-import { GEL_BREVIER } from '@bbc/gel-foundations/typography';
+import {
+  GEL_BREVIER,
+  GEL_FF_REITH_SANS,
+} from '@bbc/gel-foundations/typography';
 import { C_CLOUD_DARK } from '@bbc/psammead-styles/colours';
-import { FF_NEWS_SANS_REG } from '@bbc/psammead-styles/fonts';
 
 const StyledTimestamp = styled.span`
   ${GEL_BREVIER};
   color: ${C_CLOUD_DARK};
   display: block;
-  font-family: ${FF_NEWS_SANS_REG};
+  font-family: ${GEL_FF_REITH_SANS};
```

Comment:
```suggestion
  font-family: ${GEL_FF_REITH_SANS};
  font-weight: 500;
```
I believe everywhere you've added a `font-family`, you should have the relevant `font-weight` value used as well.

Reply:
Really? even when its not bold?

Reply:
We currently dont define a value for regular https://github.com/bbc/psammead/blob/latest/packages/utilities/psammead-styles/src/fonts.js#L4-L9 so I think we should maintain not defining one here for now

Reply:
Sounds good.  I was under the impression that Regular used the `font-weight` value equivalent of `normal` which is `500`. But that is apparently not the case.  We should separately discuss which font-weight Regular should use with UX.